src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -103,6 +103,84 @@ public InternalRequestsController(IFusionProfileResolver profileResolver)
 
         }
 
+        [HttpPost("/projects/{projectIdentifier}/resources/requests/$batch")]
+        [HttpPost("/projects/{projectIdentifier}/requests/$batch")]
+        public async Task<ActionResult<ApiResourceAllocationRequest>> CreateProjectAllocationRequestV2(
+           [FromRoute] PathProjectIdentifier projectIdentifier, [FromBody] BatchCreateResourceAllocationRequest request)
```

Comment:
I would assume that a batch endpoint should follow the format specified here:
http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_BatchRequestsandResponses